<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TriggerCommandClass</class>
 <widget class="QWidget" name="TriggerCommandClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>275</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TriggerCommand</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QCheckBox" name="trigger">
     <property name="text">
      <string>Trigger</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="container" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Statistic Type</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="statisticType">
          <item>
           <property name="text">
            <string>Single(1)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Rtd(2)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Fixed(4)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Float(5)</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="statisticCount">
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>9999</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Statistic Count</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Command lines</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="send_txt">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
